src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -63,6 +63,13 @@ namespace dsn {
 namespace replication {
 struct replica_base;
 
+DSN_DEFINE_uint64(replication,
+                  max_allowed_write_size,
```

Comment:
My intention is Reuse this parameter `max_allowed_write_size`(used play a role in `replica_2pc.cpp`). So that we do not need to configure a new parameter.

Reply:
You are using FLAGS_**dup**_max_allowed_write_size in line 246, which is not defined.

If you want to reuse max_allowed_write_size, it should be FLAGS_max_allowed_write_size, and declare like `DSN_DEFINE_uint64(max_allowed_write_size)` here.